packages/reports/tests/integration/components/report-actions/export-test.js

```diff
@@ -55,15 +52,31 @@ module('Integration | Component | report actions - export', function(hooks) {
 
     run(() => {
       let request = {
-        logicalTable: {
-          table: 'network',
-          timeGrain: 'day'
-        },
-        intervals: [
-          Store.createFragment('bard-request/fragments/interval', {
-            interval: new Interval(moment('10-02-2015', 'MM-DD-YYYY'), moment('10-14-2015', 'MM-DD-YYYY'))
-          })
-        ]
+        table: 'network',
+        dataSource: 'bardOne',
+        limit: null,
+        requestVersion: '2.0',
+        filters: [
+          {
+            type: 'timeDimension',
+            dataSource: 'bardOne',
+            field: 'network.dateTime',
+            parameters: { grain: 'day' },
+            operator: 'bet',
+            values: ['11-04-2020', '11-06-2020']
```

Comment:
Let's use the same dates as before
```suggestion
            values: ['2015-10-02', '2015-10-14']
```